### PR TITLE
smartsheets points integration script

### PR DESCRIPTION
**Pull request by matallen**

### CODE REVIEW

File: smartsheets-stats.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+
+import smartsheet,json,argparse,sys,re
+from datetime import datetime, timedelta
+from dateutil.relativedelta import relativedelta
+
+TOKEN = "7o8djelr503ekzb4phyje3wvoh"
```
Comment (sabre1041) [inline smartsheets-stats.py]: Can this be externalized?

File: smartsheets-stats.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+SHEET_ID = "3510398591756164"
```
Comment (sabre1041) [inline smartsheets-stats.py]: Same as above

File: smartsheets-stats.py [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+        if re.search("Thought Leadership.*", row["Program Name"]):
+            pool="ThoughtLeadership"
+        if re.search("Community.*", row["Program Name"]):
+            pool="ThoughtLeadership"
+        if re.search("Adopt.*", row["Program Name"]):
+            pool="ServicesSupport"
+        if re.search("First and Thirds.*", row["Program Name"]):
+            pool="ServicesSupport"
+        
+        print "{0}/SS{1}/{2}/{3} [pool={4},board=cXCGH32HjPp2mQV3MfxJX4WVVFQ9xJ5J2VCmX8F1,rowId={5},linkId={6}]".format(points_grouping, row["id"], row["Created By"],int(row["Points"]),pool,row["id"],row["Row ID"])
```
Comment (sabre1041) [inline smartsheets-stats.py]: Can the `board` variable be externalized?

**Review by sabre1041 — CHANGES_REQUESTED** [review]

File: smartsheets-stats.py (hunk 1 above)
Comment (matallen) [reply smartsheets-stats.py]: yes - done

File: smartsheets-stats.py (hunk 2 above)
Comment (matallen) [reply smartsheets-stats.py]: yes - done

File: smartsheets-stats.py (hunk 3 above)
Comment (matallen) [reply smartsheets-stats.py]: yes - done

**Review by matallen — COMMENTED** [review]
Thanks for your review as always!

**Review by sabre1041 — APPROVED** [review]
LGTM. Thanks for the updates!

**Pull request merged**